### PR TITLE
olive shared cache blog init

### PR DESCRIPTION
**Pull request by samuel100**
Added blog post on Olive's shared cache feature.

### CODE REVIEW

File: src/routes/blogs/+page.svelte [hunk 1]
```diff
@@ -45,6 +45,16 @@
 		dispatch('switchTab', tab);
 	}
 	let featuredblog = [
+		{
+			title: 'Enhancing team collaboration during AI model optimization with the Olive Shared Cache',
+			date: 'October 30th, 2024',
+			blurb:
+				"Learn how to use Olive's shared cache to enhance team collaboration when optimizing AI models",
+			link: 'blogs/olive-shared-cache',
+			image: Phi3SmallMediumImage,
```
Comment (MaanavD) [inline src/routes/blogs/+page.svelte:54]: Needs to be a blog post image (also update actual page with image)

File: src/routes/blogs/olive-shared-cache/+page.svx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    'Devang Patel',
+    'Sam Kemp'
+    ]
+authorsLink:
+  [
+    'https://www.linkedin.com/in/xiaoyu-zhang/',
+    'https://www.linkedin.com/in/devangpatel/',
+    'https://www.linkedin.com/in/samuel-kemp-a9253724/'
+   ]
+image: 'https://iili.io/d95Pwcx.png'
```
Comment (MaanavD) [inline src/routes/blogs/olive-shared-cache/+page.svx:18]: Image, same as blogs page.

File: src/routes/blogs/olive-shared-cache/+page.svx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+## 🤝 Team collaboration during optimization process
+
+User A begins the optimization process by employing Olive’s quantize command to optimize the [Phi-3-mini-4k-instruct](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct) model using the AWQ algorithm. This step is marked by the following command line execution:
+
+```bash
+olive quantize \
+  --model Microsoft/Phi-3-mini-4k-instruct \
+  --algorithm awq \
+  --account-name {AZURE_STORAGE_ACCOUNT} \
```
Comment (MaanavD) [inline src/routes/blogs/olive-shared-cache/+page.svx:41]: Maybe the format of azure storage account / storage container name can be more specific? Not a hardcore azure user so not 100% sure how I'd access - maybe links to docs?

File: src/routes/blogs/olive-shared-cache/+page.svx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+- **Resource Optimization:** Efficient use of computational power leads to better resource management, ensuring that resources are available for other critical tasks.
+- **Collaboration:** The shared cache fosters a collaborative environment where different users can benefit from each other’s optimization efforts, promoting knowledge sharing and teamwork.
+
+## Conclusion
+
+By saving and reusing optimized models, Olive’s shared cache feature paves the way for a more efficient, cost-effective, and collaborative environment. As AI continues to grow and evolve, tools like Olive will be instrumental in driving innovation and efficiency.
+Whether you are a seasoned data scientist or a newcomer to the field, embracing Olive can significantly enhance your workflow. By reducing the time and costs associated with model optimization, you can focus on what truly matters: developing groundbreaking AI models that push the boundaries of what is possible.
+Embark on your optimization journey with Olive today and experience the future of machine learning efficiency.
+
+## ⏭️ Try Olive
```
Comment (MaanavD) [inline src/routes/blogs/olive-shared-cache/+page.svx:140]: Maybe we can also add a simpler option to try olive for users who don't want to use shared cache? 

i.e. a quick auto-opt from above without account/container info?

**Review by MaanavD — COMMENTED** [review]
Other than my comments, looks great to me! After they're addressed, I think it's good to ship. 1-2 more reviews do sound like a good idea however.

Generally though, it reads incredibly clearly and is direct and to the point! Love it 💯

File: src/routes/blogs/+page.svelte (hunk 1 above)
Comment (samuel100) [reply src/routes/blogs/+page.svelte:54]: I've added the ONNX Logo. We don't have an Olive logo.

File: src/routes/blogs/olive-shared-cache/+page.svx (hunk 4 above)
Comment (samuel100) [reply src/routes/blogs/olive-shared-cache/+page.svx:140]: Added

**Review by MaanavD — APPROVED** [review]
LGTM! 🚀

**Review by devang-ml — APPROVED** [review]

**Pull request merged**